apps/studio/components/interfaces/Settings/Addons/CustomDomainSidePanel.tsx

```diff
@@ -244,19 +244,31 @@ const CustomDomainSidePanel = () => {
             <>
               {selectedOption === 'cd_none' ||
               (selectedCustomDomain?.price ?? 0) < (subscriptionCDOption?.variant.price ?? 0) ? (
-                <p className="text-sm text-foreground-light">
-                  Upon clicking confirm, the amount of that's unused during the current billing
-                  cycle will be returned as credits that can be used for subsequent billing cycles
-                </p>
+                !subscription?.billing_via_partner && (
```

Comment:
Is this correct?

Reply:
When using optional chaining, it's best to explicitly check for false or true, otherwise you'll get a false negative when the data hasn't loaded yet, i.e.

!subscription?.billing_via_partner will not render if value is false, null or undefined. During loading the data, it will be undefined, so it will show in that case, too. Best to check like

```
subscription?.billing_via_partner === false
```

Reply:
Oh yea, thanks! Changed it!

Reply:
But the condition itself is correct, right?

Reply:
Yep